README.md

```diff
@@ -77,6 +77,8 @@ Run `go install github.com/JorgeReus/aws-sso-creds/cmd/aws-sso-creds@latest`
 Usage: aws-sso-creds [-fhpst] [-r value] [-u value] [parameters ...]
  -f, --forceAuth    Force Authentication with AWS SSO
  -h, --help         Help
+     --noInteractive
```

Comment:
Replace *no* with *non* in noInteractive (i.e. nonInteractive) or even replace it with noBrowser